.nvmrc

```diff
@@ -0,0 +1 @@
+v8
```

Comment:
Put a fixed version like `8.2.1` so we all get a specific version of npm

Reply:
+1